bright-ifit-purchase-validation.yaml

```diff
@@ -147,6 +147,40 @@ components:
           type: boolean
         error:
           $ref: '#/components/schemas/ValidateSubscriptionError'
+    UserSubscriptionInfoResponse:
+      description: ''
+      type: object
+      required:
+        - accountSource
+        - sku
+        - subscriptionManagementUrl
+      properties:
+        accountSource:
+          type: string
+          enum:
+            - none
+            - icon
+            - shop
+            - amazon
+            - android
+            - firetv
+            - ios
+            - roku
+            - tvos
+            # - androidtv
```

Comment:
in the tv-app controllers, I realized I started coding this up as `androidTv`. that ok? I can make sure it's correct when I go to update this schema later

Reply:
Would it be easy enough for you to change it to all lowercase?  I just think it would fit more along with all the other `account.source` enums being all lowercase (for the sake of consistency)...except planetFitness i guess...but that value isn't actually used lol...

Reply:
tbh, I'm not gonna lose sleep over this, so I'll go with whatever you prefer 👍

Reply:
just for resolution: I [left this](lib/third-party-subscription-sources.js) as `androidTv` for now